man/systemd.network.xml

```diff
@@ -2158,6 +2158,14 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
           <para>As in the [Address] section, but defaults to true.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>RouteMetric=</varname></term>
+        <listitem>
+          <para>The metric of the route. Takes an unsigned integer in the range 0…4294967295. When
```

Comment:
"the route"? should probably say "the route to the delegated prefix subnet" or so